core/src/lib.rs

```diff
@@ -1,6 +1,8 @@
 // Mockall triggers this warning for every mocked trait. This is fixed in Mockall master but not
 // released.
 #![cfg_attr(test, allow(clippy::unused_unit))]
+// Coverage build on nightly is failing because of this
+#![allow(unused_braces)]
```

Comment:
nit: Maybe we can include this only in the module where this is happening? Specifically where we `include!(...)` the generated contract code.